security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -956,7 +955,7 @@ private HttpRequestBase makeRequest(HttpServletRequest request, String sURL) thr
 
             // handles webdav specific verbs
             String[] webdavVerb = { "COPY", "LOCK", "UNLOCK", "MKCOL", "MOVE", "PROPFIND", "PROPPATCH", "UNLOCK",
```

Comment:
* COPY does not seem to take any http entity:
https://docs.microsoft.com/en-us/previous-versions/office/developer/exchange-server-2003/aa142816(v=exchg.65)
* LOCK does
* UNLOCK does
* MKCOL does not
* MOVE does not
* PROPPATCH does
* REPORT does (https://datatracker.ietf.org/doc/html/rfc3253#section-3.7.1)

Reply:
UNLOCK is defined twice in the array

Reply:
it might work with HttpEntityEnclosingRequestBase anyway, but I think a more relevant one (the same as the base for GET requests for instance) should be used instead